agenta-cli/agenta/docker/docker-assets/Dockerfile.cloud.template

```diff
@@ -2,7 +2,7 @@ FROM public.ecr.aws/s2t9a1r1/agentaai/lambda_templates_public:main
 
 COPY requirements.txt ${LAMBDA_TASK_ROOT}
 RUN pip install --no-cache-dir --disable-pip-version-check -U agenta
-RUN pip install --no-cache-dir --disable-pip-version-check -U -r requirements.txt
+RUN pip install --no-cache-dir --disable-pip-version-check -r requirements.txt
```

Comment:
Because it  was part of the QA process. When you specify a fixed version of a package in your `requirements.txt` file, the above command will upgrade that package for you. The behaviour is wrong.